CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ The REST API versions in the table are just for your information as the values a
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="11.7.0"></a>
+## 2019-03-?? Version [11.7.0](https://github.com/gooddata/gooddata-js/compare/v11.6.0...v11.7.0)
```

Comment:
Just reminder to update a date before merge :)

Reply:
specified exact date (today)